pyscf/adc/radc.py

```diff
@@ -80,10 +80,10 @@ def kernel(adc, nroots=1, guess=None, eris=None, verbose=None):
 
 def compute_amplitudes_energy(myadc, eris, verbose=None):
 
-    t1, t2 = myadc.compute_amplitudes(eris)
+    t1, t2, t2_vvvv_imds = myadc.compute_amplitudes(eris)
     e_corr = myadc.compute_energy(t2, eris)
 
-    return e_corr, t1, t2
+    return e_corr, t1, t2, t2_vvvv_imds
```

Comment:
We shouldn't return t2_1_vvvv_imds here, just set a predefined object in the class to contain t2_1_vvvv_imds (see below).